### PR TITLE
new logos labled as rectangle.png or square.png

### PR DESCRIPTION
**Pull request by aleksa-krolls**
Following the specifications in the `template/assets` directory, @mrmatroxa has generated new app logos for all of the adaptors and labeled the files as `rectangle.png` or `square.png`. Please review and let him know if you have any feedback. 

Notes his comments in this spreadsheet re: a couple of questions and decisions taken. 
https://docs.google.com/spreadsheets/d/1zH7SAm9nfAGU7kHop0AGd0spOSRipPKHSdspL8_hx3E/edit#gid=0

### CODE REVIEW

File: packages/ocl/assets/rectangle.png
Comment (taylordowns2000) [inline packages/ocl/assets/rectangle.png:1]: Is this definitely `Roboto`? Looks a little different from the website.
image

File: packages/common/assets/square.png
Comment (taylordowns2000) [inline packages/common/assets/square.png:1]: hrmmmmmm. this is not the square image from `template`. please use that. the alpha has been removed there so there's no transparency and that's important to keep the gradient right on off-white background.

File: packages/primero/assets/rectangle.png
Comment (taylordowns2000) [inline packages/primero/assets/rectangle.png:1]: we need a dark font-color version of this instead as we'll render on many light backgrounds

File: packages/twilio/assets/square.png
Comment (taylordowns2000) [inline packages/twilio/assets/square.png:1]: different color than the rectangular version. intentional?

**Review by taylordowns2000 — CHANGES_REQUESTED** [review]
looks great. thanks @MrMatroxa . please address the review comments and ensure that no artifacts of the previous naming convention are left behind. (you mentioned changes to msgraph... you should be deleting those other files in this PR. openfn? template? googlehealthcare? make sure that the whole _repo_ looks how you want it to look and re-request a review.

thank you!!!

File: packages/primero/assets/rectangle.png
Comment (MrMatroxa) [reply packages/primero/assets/rectangle.png:1]: Do you want light AND dark text versions or just the dark one?

File: packages/ocl/assets/rectangle.png
Comment (MrMatroxa) [reply packages/ocl/assets/rectangle.png:1]: I took the font from the logo in the logo folder that I got from you. I can change it to Roboto.

File: packages/twilio/assets/square.png
Comment (MrMatroxa) [reply packages/twilio/assets/square.png:1]: Will change to the same one

File: packages/ocl/assets/rectangle.png
Comment (aleksa-krolls) [reply packages/ocl/assets/rectangle.png:1]: @taylordowns2000 how are these? https://github.com/OpenFn/adaptors/tree/new-app-logs/packages/ocl/assets

File: packages/common/assets/square.png
Comment (aleksa-krolls) [reply packages/common/assets/square.png:1]: copied over the one from the template

File: packages/primero/assets/rectangle.png
Comment (aleksa-krolls) [reply packages/primero/assets/rectangle.png:1]: @taylordowns2000 we went with this one: https://github.com/OpenFn/adaptors/blob/new-app-logs/packages/primero/assets/rectangle.png

File: packages/twilio/assets/square.png
Comment (aleksa-krolls) [reply packages/twilio/assets/square.png:1]: ✅

**Comment by aleksa-krolls** [comment]
@taylordowns2000 changes implemented, pls review and let me and @MrMatroxa know

**Review by taylordowns2000 — APPROVED** [review]

**Pull request merged**